quasar-utils/javac-plugin/src/main/java/net/corda/plugins/javac/quasar/ASTUtils.java

```diff
@@ -0,0 +1,35 @@
+package net.corda.plugins.javac.quasar;
+
+import com.sun.tools.javac.code.Scope;
+import com.sun.tools.javac.code.Symbol;
+import com.sun.tools.javac.util.Filter;
+import com.sun.tools.javac.util.Name;
+
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+
+public class ASTUtils {
+    private static final Method getTypeMembers;
+    private static final Method filterClassMembers;
+
+    static  {
+        try {
+            if (Float.parseFloat(System.getProperty("java.specification.version")) > 8) {
+                filterClassMembers = Scope.class.getMethod("getSymbolsByName", Name.class, Filter.class);
+            } else {
+                filterClassMembers = Scope.class.getMethod("getElementsByName", Name.class, Filter.class);
```

Comment:
This doesn't look brittle at all...

Reply:
this is meant to address the small differences between the jdk 8 and jdk 11 APIs

Reply:
You mean the _currently existing_ small differences, as far as JDK11.